src/constants.js

```diff
@@ -403,10 +403,6 @@ export const TASK_COMPLETION_RULE_ANY: 'ANY_ASSIGNEE' = 'ANY_ASSIGNEE';
 export const TASK_EDIT_MODE_CREATE: 'CREATE' = 'CREATE';
 export const TASK_EDIT_MODE_EDIT: 'EDIT' = 'EDIT';
 
-/* ------------------ Comment types ----------------- */
```

Comment:
To confirm: These aren't used by any consuming applications?

Reply:
Correct.